operations/configuration.md

```diff
@@ -393,31 +393,33 @@ Returns all tax environments supported by the API.
 | `Value` | number | required | Tax rate, e.g. `0.21` in case of 21% tax rate. |
 | `Strategy` | [Tax Rate Strategy](#tax-rate-strategy) | required | Tax strategy type, e.g. relative or flat. |
 
-#### Tax Rate Strategy
+#### Tax rate strategy
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `Discriminator` | [Tax Strategy Discriminator](#tax-strategy-discriminator) | required | If tax rate is flat or relative. |
```

Comment:
Also here the casing should match. And it should have string prefix. `string [Tax strategy discriminator](...)`

Reply:
line 367 threw me off as it was originally Tax Environments so I followed that. Corrected so all consequent words are lowercase throughout